clang/CMakeLists.txt

```diff
@@ -252,7 +252,7 @@ set(CLANG_SPAWN_CC1 OFF CACHE BOOL
 option(CLANG_DEFAULT_PIE_ON_LINUX "Default to -fPIE and -pie on linux-gnu" OFF)
 
 # Manually handle default so we can change the meaning of a cached default.
-set(CLANG_ENABLE_OPAQUE_POINTERS "DEFAULT" CACHE STRING
+set(CLANG_ENABLE_OPAQUE_POINTERS ${LLVM_ENABLE_OPAQUE_POINTERS} CACHE STRING
```

Comment:
It's not entirely clear to me what the upstream code is trying to accomplish with the cached default, but I think this change breaks it. Would modifying line 258 be better?

Reply:
Done, but I don't think there a practical difference for the project.